kafka-connect/kafka-connect-runtime/hive/NOTICE

```diff
@@ -1517,7 +1317,7 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-Group: org.apache.kerby  Name: kerby-asn1  Version: 1.0.1
```

Comment:
Same as above, since this is an ASF project do we even need to mention it in NOTICE?

Reply:
I added a commit with ASF projects mentions from `NOTICE`.